spec/features/ui/v1/digital_objects/metadata/edit_spec.rb

```diff
@@ -34,6 +34,35 @@
           expect(page).to have_content("Item: #{updated_title_value}")
         end
       end
+      context 'and dynamic field data is updated with a language tag' do
+        include_context 'with language subtag fixtures'
+        let(:lang_subtags) { Hyacinth::Language::SubtagLoader.new(iana_en_fixture).load }
+        let(:value_lang) { { 'tag' => 'en' } }
+        let(:updated_field_value) { "Updated Alternative Title" }
+        let(:updated_field_lang) { "sco" }
+        let(:item) { lang_subtags && FactoryBot.create(:item, :with_ascii_dynamic_field_data, uid: uid, title: title_attribute, value_lang: value_lang) }
+        before do
+          page.find('.card-header', exact_text: 'Alternative Title')
+          card_header = page.find('.card-header', exact_text: 'Alternative Title')
+          within(card_header.sibling('.card-body')) do
+            # Selenium appears to be appending to current value with fill_in, so setting to blank first
+            page.find_field('Value').set updated_field_value
+            page.find_field('Value Language').set updated_field_lang
+          end
+          find_button('Update').click
+          # do a find to make sure scripts run
```

Comment:
I don't remember what this comment means about scripts running.  Might be confusing for others too.

Reply:
The find behavior makes Capybara wait for content to be rendered by an asynch script

Reply:
You might be able to do something like this instead (not sure if I have the exact syntax right):
```
Selenium::WebDriver::Wait.new(timeout: 15).until { find_button('Update') }
```

Reply:
Well, actually, I guess if `find` waits anyway then the wait wrapper isn't adding anything.